Assets/Scripts/AI/ThirdPersonCharacter.cs

```diff
@@ -0,0 +1,225 @@
+using UnityEngine;
+
+namespace UnityStandardAssets.Characters.ThirdPerson
+{
+	[RequireComponent(typeof(Rigidbody))]
+	[RequireComponent(typeof(CapsuleCollider))]
+	[RequireComponent(typeof(Animator))]
+	public class ThirdPersonCharacter : MonoBehaviour
+	{
+		[SerializeField] float m_MovingTurnSpeed = 360;
+		[SerializeField] float m_StationaryTurnSpeed = 180;
+		[SerializeField] float m_JumpPower = 12f;
+		[Range(1f, 4f)][SerializeField] float m_GravityMultiplier = 2f;
+		[SerializeField] float m_RunCycleLegOffset = 0.2f; //specific to the character in sample assets, will need to be modified to work with others
+		[SerializeField] float m_MoveSpeedMultiplier = 1f;
+		[SerializeField] float m_AnimSpeedMultiplier = 1f;
+		[SerializeField] float m_GroundCheckDistance = 0.1f;
+
+		Rigidbody m_Rigidbody;
+		Animator m_Animator;
+		bool m_IsGrounded;
+		float m_OrigGroundCheckDistance;
+		const float k_Half = 0.5f;
+		float m_TurnAmount;
+		float m_ForwardAmount;
+		Vector3 m_GroundNormal;
+		float m_CapsuleHeight;
+		Vector3 m_CapsuleCenter;
+		CapsuleCollider m_Capsule;
+		bool m_Crouching;
```

Comment:
For all private fields, even though Unity treats access-less fields as private, you should explicitly state private. This is to stay consistent.